test/specs/reference/tests/01.reference.js

```diff
@@ -56,6 +56,14 @@ exports.execute = function (options) {
                 expect(reference._columns).toBeDefined();
             });
 
+            it('contextualize.record should return a contextualized reference object.', function() {
+                var recordReference = reference.contextualize.record;
+
+                expect(recordReference).not.toBe(reference);
+                expect(recordReference.columns.length).not.toBe(reference.columns.length);
+                expect(recordReference.columns.length).toBe(2);
```

Comment:
The test case should also check for the actual columns:
```javascript
var columns = Array.prototype.map.call(recordReference.columns, function(column){
  return column.name;
});
expect(columns).toEqual(["name", "value"])
```
Also we should check that other properties of the reference are not changed.